src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -21,13 +23,16 @@
 @Component
 public class TrustDetailsValidator {
 
-    public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext) {
+    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+
+        List<TrustDataDto> trustDataDtoList = overseasEntitySubmissionDto.getTrusts();
```

Comment:
Not sure if this is getting the trusts from the correct location, as they will initially be under update-review trusts before being copied to submission/trusts

Reply:
I think it's fine. I've essentially not changed where these are being picked up from, I'm simply now getting them from the same DTO object but a little bit later on. I can explain...